### PR TITLE
Fix/mailmap too many values unpack only

### LINKED ISSUE #60: update-lookup: crashes with "ValueError: too many values to unpack" on .mailmap parsin
```shell
(git)lena:~datalad/datalad-master[master]git
$> tributors update-lookup
INFO:allcontrib:Updating .tributors cache from .all-contributorsrc
INFO:    zenodo:Updating .tributors cache from .zenodo.json
INFO:   mailmap:Updating .tributors cache from .mailmap
Traceback (most recent call last):
  File "/home/yoh/proj/tributors/venvs/dev3/bin/tributors", line 33, in <module>
    sys.exit(load_entry_point('tributors', 'console_scripts', 'tributors')())
  File "/home/yoh/proj/tributors/tributors/client/__init__.py", line 175, in main
    main(args, extra)
  File "/home/yoh/proj/tributors/tributors/client/lookup.py", line 52, in main
    client.update_resource(resources=resources, params=extra)
  File "/home/yoh/proj/tributors/tributors/main/__init__.py", line 97, in update_resource
    resource.update_lookup()
  File "/home/yoh/proj/tributors/tributors/main/parsers/mailmap.py", line 63, in update_lookup
    self.load_data()
  File "/home/yoh/proj/tributors/tributors/main/parsers/mailmap.py", line 41, in load_data
    name, email = line.split("<")
ValueError: too many values to unpack (expected 2)
(dev3) 1 15113 ->1.....................................:Wed 18 Aug 2021 09:53:20 AM EDT:.
(git)lena:~datalad/datalad-master[master]git
$> git describe           
0.14.7-544-g379d4e2f9
changes on filesystem:                                                                                                                                                                                               
 .github/workflows/update-contributors.yml | 2 +-
(dev3) 1 15114.....................................:Wed 18 Aug 2021 09:54:31 AM EDT:.
(git)lena:~datalad/datalad-master[master]git
$> tributors --version
0.0.19
changes on filesystem:                                                                                                                                                                                               
 .github/workflows/update-contributors.yml | 2 +-
(dev3) 1 15115.....................................:Wed 18 Aug 2021 09:54:34 AM EDT:.
(git)lena:~datalad/datalad-master[master]git
$> cat .mailmap 
Alejandro de la Vega <delavega@utexas.edu>
Alex Waite <alexqw85@gmail.com> Alex Waite <alex@waite.eu>
Andy Connolly <andrew.c.connolly@dartmouth.edu> <andycon@users.noreply.github.com>
Anisha Keshavan <keshavan@berkeley.edu>
Benjamin Poldrack <benjaminpoldrack@gmail.com> <bpoldrack@users.noreply.github.com>
Christian Mönch <christian.moench@web.de> Christian Moench <christian.monch@conax.com>
Christian Olaf Häusler <der.haeusler@gmx.net> chris <chris@psyinfolaf.ovgu.de>
Dave MacFarlane <driusan@gmail.com> Dave MacFarlane <dave.macfarlane@mcin.ca>
Dave MacFarlane <driusan@gmail.com> Dave MacFarlane <driusan@mcin.ca>
Debanjum Singh Solanky <debanjum@gmail.com> Debanjum <debanjum@gmail.com> debanjum <debanjum@gmail.com>
Gergana Alteva <glalteva@gmail.com> Gergana Alteva <glalteva@users.noreply.github.com>
Jason Gors <jason.gors.work@gmail.com>
John T. Wodder II <git@varonathe.org> <jwodder@users.noreply.github.com>
Kusti Skytén <kusti.skyten@helsinki.fi> <kskyten@users.noreply.github.com>
Michael Hanke <michael.hanke@gmail.com> mih <mih>
Neuroimaging Community <committer@example.com> blah <committer@example.com> 
Neuroimaging Community <committer@example.com> <test@example.com> 
Neuroimaging Community <committer@example.com> unknown <test@nowhere.com> 
Taylor Olson <to.overlake@gmail.com> Taylor Olson <taylor.m.olson.21@dartmouth.edu>
Torsten Stoeter <torsten.stoeter@lin-magdeburg.de>
Vanessa Sochat <vsoch@users.noreply.github.com> 
Yaroslav Halchenko <debian@onerussian.com> 
Yaroslav Halchenko <debian@onerussian.com> <site-github-private@onerussian.com>
Yaroslav Halchenko <debian@onerussian.com> <yoh@smaug.cns.dartmouth.edu>
```

### PR DESCRIPTION
**Pull request by yarikoptic**
An enhanced part (no blacking, only relevant changes + orcid sample fixup) of #61 . Fixes #60

### CODE REVIEW

File: tributors/main/parsers/mailmap.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """
 
-Copyright (C) 2020 Vanessa Sochat.
+Copyright (C) 2020-2021 Vanessa Sochat.
```
Comment (vsoch) [inline tributors/main/parsers/mailmap.py]:
```suggestion
Copyright (C) 2020-2022 Vanessa Sochat.
```

File: tributors/main/parsers/mailmap.py [hunk 2]
```diff
@@ -38,9 +38,21 @@ def load_data(self):
...
+
+                # mailmap line can have more than one entry, split by right >
+                for entry in line.strip().split(">"):
+                    if not entry:
+                        continue
+                    new_name, email = map(str.strip, entry.split("<"))
+                    # only the first name matters
+                    if not name and new_name:
+                        name = new_name
+                    if not name:
```
Comment (vsoch) [inline tributors/main/parsers/mailmap.py:53]: Do we really want an error raised here?

Comment (yarikoptic) [reply tributors/main/parsers/mailmap.py:53]: Explicit better than implicit... Badly formatted file better leads to error than wrong result IMHO

**Comment by vsoch** [comment]
This looks good to me! I don't use mailmap so I think you are likely the main audience for this. This is ready to merge, but could you bump the version and add a note to the changelog?

Thanks for working on this!

**Review by vuillaut — APPROVED** [review]

**Pull request merged**